### PR TITLE
Add blog post for RAI's CPSAT

### PR DESCRIPTION
**Pull request by odow**
cc @william-macready

### CODE REVIEW

File: _posts/2022-07-12-constraint-programming.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+ 4  2  6  8  5  3  7  9  1
+ 7  1  3  9  2  4  8  5  6
+ 9  6  1  5  3  7  2  8  4
+ 2  8  7  4  1  9  6  3  5
+ 3  4  5  2  8  6  1  7  9
+```
+
+### Next-steps
+
+We still have a bit more work to go. We want to add support for reified
```
Comment (mlubin) [inline _posts/2022-07-12-constraint-programming.md:126]: It's underwhelming to say "We still have a bit more work to go" when announcing the start of the contract. How about something like "The work sponsored by RelationalAI will enable us to ..."

**Comment by william-macready** [comment]
looks good to me (especially with miles suggestions). let me confirm that we're ok being mentioned as providing financial support before you post. i

**Review by mlubin — APPROVED** [review]
LGTM from the JuMP side. Please wait for RAI approval.

**Comment by william-macready** [comment]
i got official RAI blessing so we're good to go whenever you like.

**Pull request merged**